src/lib/_utilities/i18n.utilities.test.ts

```diff
@@ -79,3 +79,113 @@ describe('Test interpolate utility function', () => {
     expect(string).toBe(expected);
   });
 });
+
+describe('Test text-to-key function', () => {
```

Comment:
The function tested here is the `textToKey` which allows us to use any unique string as a key for content lookup. It essentially converts anything thrown at it to camelCase to normalize all the keys in the locale files. 

I added as many different variations to strings I could think of to test. If you can think of others, just let me know.

Reply:
wow you got a lot of cases covered here!